components/Ticker/Ticker.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useContext, useEffect, useState } from "react"
+import { SocketIOContext } from "@/components/SocketIOProvider"
+import styles from "./Ticker.module.scss"
+
+export default function Ticker({ symbol }: { symbol: string }) {
+  const [price, setPrice] = useState(null)
+  const socket = useContext(SocketIOContext)
+
+  useEffect(() => {
+    socket.on("message", ({ price }) => {
```

Comment:
You can `import { toUSD } from "@/utils/numbers"`
and then use it to format `price` if you want